linode/manager
# Storybook can read .mdx files

## Description

- Updated Storybook dependencies to latest
- Gets Storybook to show `*.stories.mdx` files. 

## How to test

- Open Storybook and look for the Buttons story
- Make sure Manger still works (make sure I didn't mess up any webpack type of stuff)
